e2e/test-component/scenarios/embedding-sdk/editable-dashboard.cy.spec.tsx

```diff
@@ -214,6 +216,14 @@ describe("scenarios > embedding-sdk > editable-dashboard", () => {
 
         cy.log("building the query");
         H.popover().findByRole("link", { name: "Orders" }).click();
+        cy.button("Visualize").click();
```

Comment:
This asserts the back button in the visualization. The back button before the visualization is tested in unit tests.